src/Parser.js

```diff
@@ -789,7 +789,7 @@ Parser.prototype.parseSizeGroup = function(optional) {
     let res;
     if (!optional && this.nextToken.text !== "{") {
         res = this.parseRegexGroup(
-            /^[-+]? *(?:$|\d+|\d+\.\d*|\.\d*) *[a-z]{0,2}$/, "size");
+            /^[-+]? *(?:$|\d+|\d+\.\d*|\.\d*) *[a-z]{0,2}\s*$/, "size");
```

Comment:
We use `this.consumeSpaces();` in other places.  It might be good to do the same here for consistency.  It should be noted that `consumeSpaces` does not consume all whitespace, only plain old spaces.  I wonder if it should be upgrade?